pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -103,6 +103,93 @@ func (db *DB) Migration() *migrate.Migration {
 					)`,
 				},
 			},
+			{
+				// some databases may have already this done, although the version may not match
+				Description: "Optimize storagenode tables",
+				Version:     1,
+				Action: migrate.Func(func(log *zap.Logger, db migrate.DB, tx *sql.Tx) error {
+					v1sql := migrate.SQL{
+						`ALTER TABLE bandwidth_agreements RENAME TO bandwidth_agreements_old`,
+						`CREATE TABLE IF NOT EXISTS bandwidth_agreements (
+									satellite BLOB,
+									agreement BLOB,
+									signature BLOB,
+									uplink BLOB,
+									serialnum TEXT NOT NULL,
+									total INT(10),
+									maxsize INT(10),
+									createdunixsec INT(10),
+									expirationunixsec INT(10),
+									action TEXT,
+									daystartdateunixsec INT(10)
+								)`,
+						`INSERT INTO bandwidth_agreements (satellite, agreement, signature)
+								 SELECT satellite, agreement, signature FROM bandwidth_agreements_old`,
+					}
+					_ = v1sql.Run(log, db, tx)
+
+					// iterate through the table and fill
+					err := func() error {
+						rows, err := tx.Query(`SELECT * FROM  bandwidth_agreements ORDER BY satellite`)
```

Comment:
You don't need to select all columns, just need `agreement` - `SELECT agreement FROM ...`

Reply:
done